events/events.go

```diff
@@ -196,3 +204,20 @@ func (e *Events) unsubscribe(l listener) bool {
 	e.mu.Unlock()
 	return false
 }
+
+func GetContainerId(eng *engine.Engine, name string) string {
```

Comment:
I must say it's not the ideal solution to call inspect and have all these additional buffers, especially within the for loop on line 122. However, the loop is only looping through the different filters the user provided, so in practice it shouldn't be that many.